File: FridgyKeyApp/FK.TestVIEW/ClientApp/app/models/FridgeProduct.ts

export class FridgeProduct {
  public id: number;
  public passwordhash: string;
  public name: string;
  public description: string;
  //public datecreate: date; 
}  //???

export class FridgeProductView {
  public id: number;
  public fridgeid: number;
  public productid: number;
  public userid: string;
  public ei: string;
  public name: string;
  public description: string;
  public amount: number;
  public price: number; 
  //public datebuy: date; 
  //public datevalid: date; 
}

File: 000App/CoreWebApi/CoreWebApi/FK.Models/DTO/FridgeProduct.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.AspNetCore.Identity;

namespace FK.Models
{ 
    public partial class FridgeProductView
    {  
        public int Id { get; set; }
         
        public int FridgeId { get; set; }
         
        public int ProductId { get; set; }
         
        public string UserId { get; set; }
         
        public double Amount { get; set; }
         
        public string EI { get; set; }
         
        public string Description { get; set; }

        public decimal? Price { get; set; }

        public DateTime? DateBuy { get; set; }

        public DateTime? DateValid { get; set; }

        public FridgeView Fridge { get; set; }

        public ProductView Product { get; set; }

        public ApplicationUserView User { get; set; }
    }
}
